### PR TITLE
zsh-you-should-use 1.7.3 (new formula)

### PR DESCRIPTION
**Pull request by usmcfiredog**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/zsh-you-should-use.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+class ZshYouShouldUse < Formula
+  desc "ZSH plugin that reminds you to use existing aliases for commands you just typed"
+  homepage "https://github.com/MichaelAquilina/zsh-you-should-use"
+  url "https://github.com/MichaelAquilina/zsh-you-should-use/archive/1.7.3.tar.gz"
+  sha256 "db4486cd12974332ec858d446aff9393dae6be430d425a56d7036d2ce4edeb9e"
+  license "GPL-3.0-or-later"
+
+  bottle :unneeded
+
+  depends_on "zsh" => "5.1"
```
Comment (SMillerDev) [inline Formula/zsh-you-should-use.rb:10]: You shouldn't specify versions here. If it doesn't work with the latest version it shouldn't be in homebrew.

File: Formula/zsh-you-should-use.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        source #{HOMEBREW_PREFIX}/share/zsh-you-should-use/you-should-use.plugin.zsh
+
+      You will also need to force reload of your .zshrc:
+
+        source ~/.zshrc
+
+      Please check homepage for additional settings:
+        https://github.com/MichaelAquilina/zsh-you-should-use
+
+    EOS
```
Comment (SMillerDev) [inline Formula/zsh-you-should-use.rb:29]: This should be be in the documentation. Only the location of the file is homebrew specific and allowed here.

File: Formula/zsh-you-should-use.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  end
+
+  test do
+    assert_match "1.7.3",
```
Comment (SMillerDev) [inline Formula/zsh-you-should-use.rb:33]:
```suggestion
    assert_match version.to_s,
```

File: Formula/zsh-you-should-use.rb (hunk 1 above)
Comment (usmcfiredog) [reply Formula/zsh-you-should-use.rb]: I thought that's how you wrote minimum version, but I have removed it.

File: Formula/zsh-you-should-use.rb (hunk 2 above)
Comment (usmcfiredog) [reply Formula/zsh-you-should-use.rb]: removed and is in the documentation also

**Comment by usmcfiredog** [comment]
Is there anything else I need to do once I addressed the comments?

**Comment by SMillerDev** [comment]
Can you squash your commits into one matching the PR name

**Comment by usmcfiredog** [comment]
I think I did it can you let me know if it looks right now please

**Comment by SMillerDev** [comment]
Nothing changed, did you force push your changes?

**Comment by usmcfiredog** [comment]
Yes I think so

**Comment by usmcfiredog** [comment]
Ok I think I got it this time sorry

**Pull request closed without merging**